honeybee_radiance/modifierset.py

```diff
@@ -0,0 +1,702 @@
+        return self._aperture_set
+
+    @aperture_set.setter
+    def aperture_set(self, value):
+        if value is not None:
+            assert isinstance(value, ApertureSet), \
+                'Expected ApertureSet. Got {}'.format(type(value))
+            self._aperture_set = value
+        else:
+            self._aperture_set = ApertureSet()
+
+    @property
+    def door_set(self):
+        """Get or set the DoorSet assigned to this ModifierSet."""
+        return self._door_set
+
+    @door_set.setter
+    def door_set(self, value):
+        if value is not None:
+            assert isinstance(value, DoorSet), \
+                'Expected DoorSet. Got {}'.format(type(value))
+            self._door_set = value
+        else:
+            self._door_set = DoorSet()
+
+    @property
+    def modifiers(self):
+        """List of all modifiers contained within the set."""
+        return self.wall_set.modifiers + \
+            self.floor_set.modifiers + \
+            self.roof_ceiling_set.modifiers + \
+            self.aperture_set.modifiers + \
+            self.door_set.modifiers
+
+    @property
+    def modified_modifiers(self):
+        """List of all modifiers that are not defaulted within the set."""
+        return self.wall_set.modified_modifiers + \
+            self.floor_set.modified_modifiers + \
+            self.roof_ceiling_set.modified_modifiers + \
+            self.aperture_set.modified_modifiers + \
+            self.door_set.modified_modifiers
+
+    @property
+    def unique_modifiers(self):
+        """List of all unique modifiers contained within the set."""
+        return list(set(self.modifiers))
+
+    @property
+    def unique_modified_modifiers(self):
+        """List of all unique modifiers that are not defaulted within the set."""
+        return list(set(self.modified_modifiers))
+
+    def get_face_modifier(self, face_type, boundary_condition):
+        """Get a modifier object that will be assigned to a given type of face.
+
+        Args:
+            face_type: Text string for the type of face (eg. 'Wall', 'Floor',
+                'Roof', 'AirWall').
+            boundary_condition: Text string for the boundary condition
+                (eg. 'Outdoors', 'Surface', 'Adiabatic', 'Ground')
+        """
+        if face_type == 'Wall':
+            return self._get_modifier_from_set(self.wall_set, boundary_condition)
+        elif face_type == 'Floor':
+            return self._get_modifier_from_set(self.floor_set, boundary_condition)
+        elif face_type == 'RoofCeiling':
+            return self._get_modifier_from_set(self.roof_ceiling_set, boundary_condition)
+        elif face_type == 'AirWall':
+            return air_wall
+        else:
+            raise NotImplementedError(
+                'Face type {} is not recognized for ModifierSet'.format(face_type))
+
+    def get_aperture_modifier(self, boundary_condition, aperture_type, parent_face_type):
```

Comment:
I am glad to see that we are using the same logic here as `honeybee-energy`.  I also see the acknowledgement of the `OperableWindow` aperture type, making me more confident that it should be moved to the core.

Reply:
let's chat about this. I think you have a point and the operable window type started to be a challenge. I think it will be even more of a challenge when we want to also handle CFD models. If you think moving it to honeybee-core will be helpful then I agree that we should do it.

And yes. I tried to keep the API consistent with `honeybee-energy` between the two so we can reuse it between the libraries.